ci/scripts/run_benchmarks.sh

```diff
@@ -37,6 +37,9 @@ RESULTS_DIR=$(pwd)/results/benchmarks-${CLUSTER_TAG}
 CLUSTER_COUNT=4
 BENCHMARKS_BRANCH=${BENCHMARKS_BRANCH:-develop}
 
+GEODE_REPO=${GEODE_REPO:-apache/geode}
```

Comment:
Are changes needed in the rest of the CI to get the `GEODE_REPO` passed through?

Reply:
see my change on line 80

Reply:
for the public Geode develop and release pipelines, the default initialized on line 40 is sufficient.  As described in the PR, if you are trying to do any testing against a PR, fork, etc prior to merging to develop, the hardcoded defaults in run_against_baseline made it impossible without hacking some scripts.   This PR eliminates the need for hacking, by improving the default for GEODE_REPO, and further allowing it to be overridden externally if the default computed on line 40 doesn't work for some reason.